src/maplibre_helpers.ts

```diff
@@ -265,6 +296,10 @@ const layerZorder = [
   "route-lines",
   "route-polygons",
 
+  // Display optional layers on top of interventions
```

Comment:
I don't know if this is the right choice yet. If there's a school on top of a big area and you hover, currently you'll get both popups. Need to think more and get feedback about how this contextual data will be used in decision-making.